apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -0,0 +1,219 @@
+import {
+    IncidentWhereInput,
+    Incident as IIncident,
+    IncidentClassifierIncidentWhereInput,
+    IncidentStatusType,
+    IncidentClassifier as IIncidentClassifier,
+    IncidentClassifierWhereInput,
+} from '@app/condo/schema'
+import { Col, Row } from 'antd'
+import dayjs from 'dayjs'
+import get from 'lodash/get'
+import React, { ComponentProps, useCallback, useEffect, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography, Alert } from '@open-condo/ui'
+
+import {
+    IncidentProperty,
+    Incident,
+    IncidentClassifierIncident,
+} from '@condo/domains/ticket/utils/clientSchema'
+
+
+type Gutters = ComponentProps<typeof Row>['gutter']
+
+const INCIDENTS_GUTTER: Gutters = [0, 24]
+const DESCRIPTION_GUTTER: Gutters = [0, 14]
+
+type classifierDataType = Pick<IIncidentClassifier, 'category' | 'problem'>
+
+type IncidentHintsProps = {
+    propertyId: string
+    organizationId: string
+    classifier?: classifierDataType
+    dateISO?: string
+    onlyActual?: boolean
+}
+
+const formatDate = (date?: string) => {
+    if (!date) return '…'
+    return dayjs(date).format('DD.MM.YYYY HH.mm')
+}
+
+export const IncidentHints: React.FC<IncidentHintsProps> = (props) => {
+    const intl = useIntl()
+    const MoreLabel = intl.formatMessage({ id: 'incident.hints.more.label' })
+
+    const { propertyId, dateISO, classifier, organizationId, onlyActual = false } = props
+
+    // Logic:
```

Comment:
I suppose it's better to move this comment out of the component code. You can place it at the docstring section